platforms/software/testing-base/src/main/java/org/gradle/api/internal/tasks/testing/worker/ForkingTestClassProcessor.java

```diff
@@ -110,7 +110,6 @@ RemoteTestClassProcessor forkProcess() {
         builder.applicationClasspath(classpath.getApplicationClasspath());
         builder.applicationModulePath(classpath.getApplicationModulepath());
         options.copyTo(builder.getJavaCommand());
-        builder.getJavaCommand().jvmArgs("-Dorg.gradle.native=false");
```

Comment:
🤔 I'd love to understand why it was there in the first place.

Reply:
It seemst to improve performance a bit: https://github.com/gradle/gradle/pull/1883

Reply:
We need to figure out how we can disable loading native services in test executor without a system property.